docs/src/tutorials/gnn_intro_pluto.jl

```diff
@@ -4,6 +4,22 @@
 using Markdown
 using InteractiveUtils
 
+# ╔═╡ 6f20e59c-b002-4d22-9ee0-b62596574776
+begin
+    using Pkg
+    Pkg.activate(; temp=true)
+    packages = [
+        PackageSpec(; path=joinpath(@__DIR__,"..","..","..")),
+        PackageSpec(; name="Flux", version="0.13"),
+		PackageSpec(; name="MLDatasets", version="0.7"),
+		PackageSpec(; name="GraphMakie"),
+		PackageSpec(; name="Graphs"),
+		PackageSpec(; name="CairoMakie"),
+		PackageSpec(; name="PlutoUI"),
+    ]
+    Pkg.add(packages)
+end
```

Comment:
@rikhuijzer a problem with this is that I get a very long terminal output in the notebook that I don't know how to suppress

Reply:
maybe I just add all those packages to `docs/Project.toml` and activate that?

Reply:
In Pluto you mean right? In the few latest versions you can disable console output per cell. Select it at the top right of the cell. It remembers your choice also between runs

Reply:
And if you want to hide the code for a cell, use `# hideall` like in Documenter:

```julia
# hideall
x = 1
```
will show only the output block containing 1. To hide code and output, use the semicolon.

Reply:
what happens with the visibility toggle in Pluto frontend? Isn't the visibility state stored in the cell metadata?

Reply:
Yep nowadays it is. I have to implement reading that still. Added it to my task list a few minutes ago (https://github.com/rikhuijzer/PlutoStaticHTML.jl/issues/119)

Reply:
> Isn't the visibility state stored in the cell metadata?

I've tested this now but apparently not no in 0.19.5